scripts/update-appup.sh

```diff
@@ -16,12 +16,15 @@ cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")/.."
 PROFILE="${1:-}"
 case "$PROFILE" in
     emqx-ee)
+        GIT_REPO='emqx/emqx-enterprise.git'
         TAG_PREFIX='e'
         ;;
     emqx)
+        GIT_REPO='emqx/emqx.git'
         TAG_PREFIX='v'
         ;;
     emqx-edge)
+        GIT_REPO='emqx/emqx'
```

Comment:
Nit: maybe this should end in `.git`, so it won't return the enterprise remote later in the script?